app/dao/dao_probability_method.py

```diff
@@ -0,0 +1,156 @@
+import mysql.connector
+
+
+from parameters import HOST, USER, PASSWORD, DATABASE
+
+
+
+def conect_database(host, user, password, database):
+
+    """Essa função tem como objetivo se conectar
+    com o banco de dados"""
+
+    connetion= mysql.connector.connect(
+        host=host,
+        user=user,
+        password=password,
+        database=database,
+
+    )
+    cursor = connetion.cursor(dictionary=True)
+
+    return connetion, cursor
+
+
+def get_user_questions(id_user: int):
+    connection, cursor = conect_database(
+        host=HOST,
+        user=USER,
+        password=PASSWORD,
+        database=DATABASE
+    )
+
+    query = f"""select * from `user` as u
+                left join weather_option as wo on u.id = wo.id_user
+                left join travel_activities as ta on u.id = ta.id_user
+                left join travel_options as t_o on u.id = t_o.id_user
+                left join travel_cultures as tc on u.id = tc.id_user
+                where u.id = {id_user};"""
+    
+    try:
+        cursor.execute(query)
+        result = cursor.fetchone()
+    except Exception:
+        connection.close()
+        return False
+    else:
+        connection.close()
+        return result
+    
+
+def get_travels():
+    connection, cursor = conect_database(
+        host=HOST,
+        user=USER,
+        password=PASSWORD,
+        database=DATABASE
+    )
+
+    query = f"""select * from accommodation;"""
+    
+    try:
+        cursor.execute(query)
+        result = cursor.fetchall()
+    except Exception:
+        connection.close()
+        return False
+    else:
+        connection.close()
+        return result
+    
+
+def get_tour_travel(id_accommodation: int):
+    connection, cursor = conect_database(
+        host=HOST,
+        user=USER,
+        password=PASSWORD,
+        database=DATABASE
+    )
+
+    query = f"""select * from tour where id_accommodation = {id_accommodation};"""
```

Comment:
Poderiamos juntar o retorno da chamada de transport e tour 
`select to*, tr.* from accommodation as ac join tour as to on to.id_accommodation = {id_accommodation} join transport as tr on tr.id_accommodation = {id_accommodation}  where ac.id = {id_accommodation};`

Reply:
commitei o pullrequest com a adição do select join